### PR TITLE
stability: add blockWriter case for inserting data 

### PR DESCRIPTION
**Pull request by cwen0**
@weekface @zyguan @xiaojingchen @shuijing198799 PTAL

### CODE REVIEW

File: tests/pkg/blockWriter/blockWriter.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+	wg.Add(1)
+	go c.generateQuery(ctx, queryChan, &wg)
+
+	for {
+		select {
+		case <-c.stopChan:
+			glog.Infof("[%s] stoping...", c)
+			cancel()
+			wg.Wait()
```
Comment (weekface) [inline tests/pkg/blockWriter/blockWriter.go]: add `wg.Wait()` to this block is strange.

**Comment by weekface** [comment]
@cwen0 fix the conflicts.

**Comment by cwen0** [comment]
@weekface PTAL again, thanks!

**Comment by weekface** [comment]
@xiaojingchen PTAL

**Pull request merged**